google-sheets/functions/check-sheets-config.js

```diff
@@ -0,0 +1,47 @@
+const { google } = require('googleapis');
```

Comment:
due to the sensitive data in this and the `log-sms` files, we should probably make these either private or protected in order to be called. Ideally they would be both private, but my guess is `check-sheets-config` might need to be protected as it's called on the index page

This might require some additional error handling on the index page for this if an unauthorized user attempted to call `check-sheets-config`

Reply:
Protected won't work because you can't generate a valid signature from a front-end. The only way you can "protect" it is by making it public and then using a user specified token like we do with the other apps.